src/elements/content-uploader/ContentUploader.js

```diff
@@ -580,12 +580,16 @@ class ContentUploader extends Component<Props, State> {
             }
         }
 
-        this.updateViewAndCollection(updatedItems, itemUpdateCallback);
+        this.updateViewAndCollection(updatedItems, () => {
+            if (itemUpdateCallback) {
+                itemUpdateCallback();
```

Comment:
I'm not sure if this is needed, but if we are being more defensive around `itemUpdateCallback` you may also want to ensure that it is a function as well before calling it.

Reply:
The flow type of it is `Function` so I think it can only be a real function or `undefined`.